src/poll.rs

```diff
@@ -1161,7 +1161,7 @@ impl Poll {
         self.readiness_queue.poll(&mut events.inner);
 
         // Return number of polled events
-        Ok(events.len())
+        Ok(events.iter().count())
```

Comment:
Yeah, we should definitely not return the number... We have to leave it for now, but that return value should be deprecated.